src/Commands/Modules/Utility/CollaborationCommand.cs

```diff
@@ -38,7 +38,7 @@ public async Task StartCollabDialogueAsync()
             else
             {
                 CollabService.DeactivateUser(Context.User);
-                await Context.Channel.SendMessageAsync("Process cancelled. You can run the command again to restart the process.");
+                await Context.Channel.SendMessageAsync("Process cancelled.");
```

Comment:
It just seemed like unnecessary clutter. Running the command again will of course run the command again. Also since now the process can be restarted in DMs without running the command again it's a bad thing to suggest people to use it that way.

Reply:
Makes sense. Fair enough